docs/content/docs/operations/configuration.md

```diff
@@ -22,8 +22,261 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-{{< hint warning >}}
-**TODO**: How to configure Flink Agents for local execution and Flink cluster execution.
+## How to configure Flink Agents
 
-**TODO**: List of all built-in Configuration options.
-{{< /hint >}}
\ No newline at end of file
+There are three ways to configure Flink Agents:
+
+1. **Explicit Settings in Code**
+2. **YAML Configuration File**
+3. **Configuration During Build Agent**
+
+{{< hint info >}}
+The priority of configuration sources, from highest to lowest, is: **Explicit Settings**, followed by **YAML Configuration File**, and finally **Configuration During Build Agent Time**. In case of duplicate keys, the value from the highest-priority source will override those from lower-priority ones.
```

Comment:
I think configuration during building agent time should have the highest priority.